app/routes.py

```diff
@@ -0,0 +1,292 @@
+from flask import Blueprint, jsonify, make_response
+from flask.globals import request
+from app.models.customer import Customer 
+from app.models.video import Video
+from app.models.rental import Rental, due_date
+from app import db
+from datetime import date, datetime
+
+from tests.test_wave_02 import CUSTOMER_ID
+
+
+# --------------------------------
+# -------- CUSTOMER ROUTES -------
+# --------------------------------
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+
+    missing = ""
+    if "name" not in request_body:
+        missing = "name"
+    elif "postal_code" not in request_body:
+        missing = "postal_code"
+    elif "phone" not in request_body:
+        missing = "phone"
+    if missing:
+        return {"details": f"Request body must include {missing}."}, 400
```

Comment:
Nice validation! You might be able to make this a bit less repetitive by passing in a tuple of required elements to loop over. i.e.

```
for param_name in param_names:
    if param_name not in request_body:
        return {"details": f"Request body must include {param_name}."}, 400
```